plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1125,15 +1125,10 @@ public void testJoinPushdownDisabled()
                 .setSystemProperty("optimize_hash_generation", "false")
                 .build();
 
-        PlanMatchPattern partitionedJoinOverTableScans = node(JoinNode.class,
-                exchange(ExchangeNode.Scope.REMOTE, ExchangeNode.Type.REPARTITION,
-                        node(TableScanNode.class)),
-                exchange(ExchangeNode.Scope.LOCAL,
-                        exchange(ExchangeNode.Scope.REMOTE, ExchangeNode.Type.REPARTITION,
-                                node(TableScanNode.class))));
-
         assertThat(query(noJoinPushdown, "SELECT r.name, n.name FROM nation n JOIN region r ON n.regionkey = r.regionkey"))
-                .isNotFullyPushedDown(partitionedJoinOverTableScans);
+                .isNotFullyPushedDown(node(JoinNode.class,
```

Comment:
Was it significant that it was a partitioned join instead of some other type of join?

Reply:
I don't know. we would need to ask the author of that code -- https://github.com/trinodb/trino/commit/d7c825ce3fb5b8a16ec6eb417503c728e43282cb

Reply:
I'd actually say we should ask the reviewers - I'm surprised no-one asked about it during review. It must've been something obvious. 😆